### PR TITLE
Hiiq alarm

### PR DESCRIPTION
**Pull request by AshakaE**
Search addresses from env and return hiiq status
image
based on threshold, for now it's less than 40M

closes https://github.com/EveripediaNetwork/issues/issues/480

### CODE REVIEW

File: src/events/common.ts [hunk 1]
```diff
@@ -39,18 +39,44 @@ export class AppDiscord {
...
+      await this.updates.sendUpdates({
+        channelId: prodWikiChannel,
+        channelType: ChannelTypes.PROD,
+        url: `${this.PROD_URL}`,
+        updateType: UpdateTypes.WIKI,
+      })
+    })
 
+    // #TODO: check every one hr for hiiqAlarm
+    // #FIXME: Set back time to 1hr interval
```
Comment (kesar) [inline src/events/common.ts]: i guess you need to fix this

**Comment by kesar** [comment]
doesnt it need an env.sample with how this work? I am not sure where you put the thresholds limits

File: src/utils/sendUpdates.ts [hunk 2]
```diff
@@ -13,51 +22,66 @@ export default class Updates {
...
     return exampleEmbed
   }
+  private async messageHiiqStyle(iq: HiiqResult) {
+    const address = Object.keys(iq)[0]
+    const content = Object.values(iq)[0]
+    console.log(address)
+    const exampleEmbed = this.messageEmbed
+      .setColor(content.alarm ? '#ff0000' : '#00ff00')
+      .setTitle(content.alarm ? 'Hiiq Low' : 'Hiiq High')
+      .setDescription(`value ${content.result}`)
```
Comment (kesar) [inline src/utils/sendUpdates.ts]: convert that to IQ w ethers.formatEther ( https://docs.ethers.io/v4/api-utils.html )

**Comment by AshakaE** [comment]
> that limit is different in each contract needs to be in the configuration

like IQ(40M) and Frax(6M) ?

**Comment by kesar** [comment]
> > that limit is different in each contract needs to be in the configuration
> 
> like IQ(40M) and Frax(6M) ?

its in the issue description

File: src/events/common.ts [hunk 3]
```diff
@@ -39,18 +39,41 @@ export class AppDiscord {
...
-      await this.updates.sendUpdates(prodChannel, ChannelTypes.PROD, this.PROD_URL)
+      await this.updates.sendUpdates({
+        channelId: prodWikiChannel,
+        channelType: ChannelTypes.PROD,
+        url: `${this.PROD_URL}`,
+        updateType: UpdateTypes.WIKI,
+      })
+    })
 
+    schedule.scheduleJob('* * *', async () => {
```
Comment (kesar) [inline src/events/common.ts]: isnt this each second? i am not sure bcs of three *, just make sure its tested 👍🏻 we dont want to get banned from etherscan or discord, it should be per hour 👍🏻

File: src/events/common.ts [hunk 4]
```diff
@@ -67,7 +67,7 @@ export class AppDiscord {
       })
     })
 
-    schedule.scheduleJob('* * *', async () => {
+    schedule.scheduleJob('*/1 * * *', async () => {
```
Comment (kesar) [inline src/events/common.ts:70]: have u tested it? still looks weird

Comment (AshakaE) [reply src/events/common.ts:70]: 😂 I know, it should work though ... I tested for every minute which is `*/1 * * * *`, per the docs 1hr will be ``*/1 * * *``

**Comment by kesar** [comment]
👍

**Review by kesar — APPROVED** [review]

**Pull request merged**